validation-test/SILOptimizer/large_nested_array.swift.gyb

```diff
@@ -0,0 +1,27 @@
+// RUN: %empty-directory(%t)
+// RUN: %gyb %s > %t/main.swift
+
+// The compiler should finish in less than 2 minautes. To give some slack,
+// specify a timeout of 4 minutes.
+// If the compiler needs more than 5 minutes, there is probably a real problem.
+// So please don't just increase the timeout in case this fails.
+
+// RUN: %{python} %S/../../test/Inputs/timeout.py 240 %target-swift-frontend -O -parse-as-library -sil-verify-none -c %t/main.swift -o %t/main.o
```

Comment:
This should really be a scale test, no?

Reply:
Well, this is the end-to-end test.
But it's probably worth adding a scale-test, too. I'll take a look.